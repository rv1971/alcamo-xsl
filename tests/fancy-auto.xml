<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc
    xmlns:dc="http://purl.org/dc/terms/"
    xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/fancy.xsl"/>

    <xsl:template match="/">
      <a:result>
        <xsl:apply-templates select="/a:doc/a:data"/>
      </a:result>
    </xsl:template>

    <xsl:template match="a:item">
      <a:item>
        <xsl:value-of select="@*"/>
        <xsl:value-of select="' '"/>
        <xsl:apply-templates select="@*" mode="a:auto"/>
      </a:item>
    </xsl:template>
  </xsl:stylesheet>

  <a:data>
    <a:item foo="bar"/>
    <a:item dc:created="2023-04-17T14:48:05+02:00"/>
    <a:item dc:created="2023-04-17T14:48:13+02:00"/>
  </a:data>
</a:doc>
