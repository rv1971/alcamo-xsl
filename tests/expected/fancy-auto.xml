<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <a:item>bar bar</a:item>
  <a:item>2023-04-17T14:48:05+02:00 2023-04-17 14:48:05+02:00</a:item>
  <a:item>2023-04-17T14:48:13+02:00 2023-04-17 14:48:13+02:00</a:item>
</a:result>
